MANIFEST.in

```diff
@@ -14,3 +14,4 @@ exclude .travis.yml
 global-exclude .git*
 global-exclude __pycache__/*
 global-exclude **/.hypothesis*
+global-exclude *.py[co]
```

Comment:
There are `*.pyc` files generated. Is it a problem when they are excluded from sdist?
```
  flake8_rst/__init__.pyc
  flake8_rst/application.pyc
  flake8_rst/checker.pyc
  flake8_rst/rst.pyc
  flake8_rst/sourceblock.pyc
```

Reply:
I've no idea what in the last commit caused python to generate those files.

Reply:
pyc files compiles form python2, i think this is not a problem.